src/TaskManager/Plug-ins/Argo/ArgoProvider.cs

```diff
@@ -27,26 +27,34 @@ namespace Monai.Deploy.WorkflowManager.TaskManager.Argo
     public class ArgoProvider : IArgoProvider
     {
         private readonly ILogger<ArgoProvider> _logger;
-        private readonly IHttpClientFactory _httpClientFactory;
+        private readonly IHttpClientFactory _clientFactory;
 
         public ArgoProvider(ILogger<ArgoProvider> logger, IHttpClientFactory httpClientFactory)
         {
             _logger = logger ?? throw new ArgumentNullException(nameof(logger));
-            _httpClientFactory = httpClientFactory ?? throw new ArgumentNullException(nameof(httpClientFactory));
+            _clientFactory = httpClientFactory;
```

Comment:
Any reason to remove the null check here?

Reply:
no, that was a mistake, will put it back